ocaml/networkd/networkd_db/dune

```diff
@@ -2,6 +2,7 @@
  (name networkd_db)
  (public_name networkd_db)
  (package xapi-networkd)
+ (modes byte exe)
```

Comment:
We probably don't want the `byte` modes, we only deploy the native executables, but it doesn't really hurt to have them here (as long as we don't accidentally end up with a bytecode linked with ocamlrun instead of an ocamlopt compiled one, but we can check for that by running `strings -a` on the binary and looking for `caml_interp` I think)